src/groups.py

```diff
@@ -34,8 +34,12 @@ def expand_nodes(node_list):
     del nodes[-1]
     return nodes
 
-def __nodeattr(file_path='{}genders'.format(config.LEADER), arguments=[], split_char="\n"):
+def __nodeattr(file_path='{}genders'.format(config.LEADER),
+               arguments=[],
+               split_char="\n"):
     try:
         return local['nodeattr']['-f', file_path](arguments).split(split_char)
+    # in the case that the genders file can't be accessed, we're acting as if
+    # there's an empty genders file in that location
     except ProcessExecutionError as e:
-        raise ClickException(e.stderr.rstrip())
```

Comment:
Restore this exception to its original format as it will catch an invalid genders file (as opposed to a missing one)

Reply:
https://github.com/alces-software/adminware/pull/169/commits/79982bef3b7f4c4157ea2cd375bd0a81f2392dfe